app/views/settings/blocks/index.haml

```diff
@@ -25,7 +25,10 @@
         %li.list-group-item
           .d-flex
             %div
-              %p.mb-0= block.question.content
+              - if block.question.present?
+                %p.mb-0= block.question.content
+              - else
+                %p.mb-0.text-muted.font-italic= t('.deleted_question')
```

Comment:
Also use double quotes in HAML please! (see the other `t()` calls in the same template)